portfolio/src/main/java/com/google/sps/servlets/CommentsServlet.java

```diff
@@ -49,11 +49,26 @@ public void doGet(HttpServletRequest request, HttpServletResponse response) thro
     DatastoreService datastore = DatastoreServiceFactory.getDatastoreService();
     PreparedQuery commentsPrepared = datastore.prepare(commentsQuery);
 
-    // Loop through each Comment entity and store in an ArrayList.
+    // Get maxComments query and check it is valid.
```

Comment:
This is probably unnecessary to state, as the code is clear enough on its own. I recommend deleting this comment.